### DISCUSSION TITLE
Help with Overshoot Issue in Versatile Thermostat

### DISCUSSION THREAD

**Discussion by Mels-V** [jmcollin78/versatile_thermostat#520] [Q&A] [answered]
Hi Versatile Thermostat users,

I’m new to VTherm and currently using only the default settings. My house isn’t well insulated, so I’d rather have undershoot than overshoot. Unfortunately, I sometimes experience significant overshoots—up to 2 degrees—on some thermostats. Could someone point me in the right direction for adjusting this? I’m familiar with Home Assistant but can’t seem to figure out why I’m getting such large overshoots.

Any advice would be greatly appreciated!

Thanks,
Mels

**Comment by jmcollin78** [comment, marked as answer]
Hello @Mels-V ,

I need more informations: what kind of VTjherm ? What kind of underlying device ? What is your VTherm configuration (dev Tools / States). Without this informations, I can't help.

Please read this: https://github.com/jmcollin78/versatile_thermostat/blob/main/README.md#the-radiator-heats-up-even-though-the-setpoint-temperature-is-exceeded-or-does-not-heat-up-even-though-the-room-temperature-is-well-below-the-setpoint

and this : https://github.com/jmcollin78/versatile_thermostat/blob/main/README.md#the-radiator-heats-up-even-though-the-setpoint-temperature-is-exceeded-or-does-not-heat-up-even-though-the-room-temperature-is-well-below-the-setpoint 

and give some regulation curves.

**Comment by Mels-V** [comment]
Hello @jmcollin78 

Thanks a lot for your quick reply, and I apologize for the lack of detail in my previous post. The earlier message was a bit of a scream for help, haha!

Let me provide the necessary details. I’m currently running a few rooms with electric heaters, and my house is poorly insulated compared to the typical standards here in the Netherlands. I’m unsure what norms the default values in VTherm are based on, so maybe that’s contributing to the issue.

I’m using a central configuration with both motion detection and presence detection enabled.

In this images, you can see the room temperature exceeded the target by quite a bit. It looks like the heater activated even though the current temperature was already well above the target temperature.
![image](https://github.com/user-attachments/assets/5c992d2c-8ce6-4cce-a142-a7f1f0c5654d)
![image](https://github.com/user-attachments/assets/e30f6c0a-b8c5-48c1-bed3-37bce064a535)

For my thermostats, I’m using the central control for everything except the presets. Here are the central control parameters:
![image](https://github.com/user-attachments/assets/f21fcc7c-ae80-4da3-9588-7d68ea2ec95f)
![image](https://github.com/user-attachments/assets/7b2306d3-da01-44f8-b365-0364691bc9e6)

I really appreciate your help with this! 😊

Best regards,
Mels

**Comment by jmcollin78** [comment]
Hello,  do you have read the link I give you ? I think not because all answers are there:

![Capture d’écran 2024-10-05 à 12 09 25](https://github.com/user-attachments/assets/6407864b-5d14-4b4a-9c0f-d724e223777c)

And 

![Capture d’écran 2024-10-05 à 12 10 51](https://github.com/user-attachments/assets/dbf3146c-ee34-43a5-9a6b-3db2a77970c3)

In your case, coef_ext is too high.  Try ith something like 0.005 (there is a bug in HA UI which prevent you to type this number. You have to copy/paste it)
